js/Pane.js

```diff
@@ -77,11 +89,14 @@ var classNames = require('classnames');
     return (
       <div className={windowClassNames}
         onClick={this.focus}
+        onDoubleClick={this.reset}
+        onWheel={this.zoom}
         ref={(ref) => this._windowRef = ref}>
         <div className={barClassNames}
           ref={(ref) => this._barRef = ref}>
           <button title="close" onClick={this.close}>X</button>
           <button title="save" onClick={this.download}>&#8681;</button>
+          <button title="reset" onClick={this.reset} hidden={!this.props.handleReset}>&#10226;</button>
```

Comment:
Ah this is great! I can imagine other panes might use this reset feature somewhere down the road. 👍